files/clappr-rtmp/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "clappr-rtmp-plugin",
+  "repo": "flavioribeiro/clappr-rtmp-plugin",
+  "files": {
+    "basePath": "dist/",
+    "include": ["rtmp.min.js", "rtmp.js", "./assets/**/*"]
```

Comment:
I see, can you then remove `rtmp.js` as well? The min is enough for production I think